### DISCUSSION TITLE
Cockpit Door Video or Jpeg?

### DISCUSSION THREAD

**Discussion by MjrIncident** [flybywiresim/aircraft#5919] [Ideas]
Is this video or a jpeg?
If it was video wouldn't we see people outside the cabin door doing stuff?
I never see any people even though I have boarded passengers.

**Comment by tracernz** [comment]
It's a static image.

**Reply by CptBayer** [reply]
... have you seen the people waking down the gangway? So you know why you don´t see people at any time in the door cam ... :-p ...

joking aside ...it would be a nice gimick if after pressing the fwd call button a stewardess or similar will knock on the door and ask what u want. simultaneously another static picture is available for the cam with a person standing in front of the door :-) ...

greets
